### ISSUE TITLE
Feature request: RSS feed for news

### ISSUE THREAD

**Issue by Mrnoname1000** [bedrocklinux/bedrocklinux-website#33]
Would it be possible to add an RSS feed? I like to keep up with the news, and checking the website every time is cumbersome.

**Comment by paradigm** [comment]
In my experience, many websites don't explicitly list RSS support, but have it if you look at their source.  For example, neither "rss" nor "atom" show up in the browser-rendered https://news.ycombinator.com but if you look at its source you'll find

```
<link rel="alternate" type="application/rss+xml" title="RSS" href="rss">
```

I suspect the intent here is to feed the page's URL into RSS feed readers which parse out that bit.

The Bedrock website currently follows this same convention.  The browser rendered https://bedrocklinux.org does not explicitly mention rss or atom, but in its source you'll find:

```
<link rel=alternate type=application/atom+xml title="Bedrock Linux Feed" href=./atom.xml>
```

Web server logs indicate quite a lot of the traffic to the website is from RSS feed readers, so others did figure this out.

All that having been said, I'm certainly amenable to making it more accessible.  Where do you have in mind that would make more sense?

**Comment by Mrnoname1000** [comment]
I imagine something simple like a button at the top of the news page. For example, the button at the top of [Dolphin's blog](https://dolphin-emu.org/blog).

**Comment by paradigm** [comment]
Seems reasonable enough.  I'll look into it when I get the chance.

**Comment by Bobbbay** [comment]
I'll also input that a podcast can use RSS feeds to publish on well-known networks such as Google Podcasts, iTunes / Apple Podcasts, Stitcher, Spotify, TuneIn, etc. I know a podcast might seem a bit far off but 20 minutes a week seems definitely possible for a project this large! Would also attract some audience, though I know you aren't specifically pushing for that at the moment.

I would listen to a Bedrock Podcast, as well as read a blog. My two cents :)

**Comment by paradigm** [comment]
A podcast would be very cool.  My gut instinct is that it's unlikely to happen any time soon, as:

- If I have time available for such things, I'd probably be more interested in using it to work on Bedrock's open R&D problems.
- If I make any regularly recurring Bedrock content, it'd probably be something closer to (amusingly enough in this context) Dolphin's blog, listing out technical efforts I've been working on.

That having been said, it's not impossible the R&D part of the project eventually settles, after which there are natural podcast topics like getting X feature to work across distros or highlighting how getting Y feature from some distro is useful.

**Comment by Bobbbay** [comment]
Yeah, that I completely understand :) I wanted to mention progress reports would be interesting, but saw it was already here 😄 

Anyways, I highly doubt r&d will stop for Bedrock, which is just good news 😆. 

i.e. I, as well as many others, would love a progress blog. As for a podcast, time is the most valuable thing a man can spend...
